rollup.config.js

```diff
@@ -32,9 +31,6 @@ export default (outputFile, overrides = {}) => ({
     // which external modules to include in the bundle
     // https://github.com/rollup/rollup-plugin-node-resolve#usage
     resolve(),
-
-    // Resolve source maps to the original source
-    sourceMaps(),
```

Comment:
What's the alternative? It can be useful to have this working

Reply:
i couldn't find an alternative & this package isn't being maintained regularly so it looks like if we want the functionality we have to create our own plugin at the moment